src/Voron/StorageEnvironment.cs

```diff
@@ -391,7 +392,7 @@ internal void TransactionAfterCommit(LowLevelTransaction tx)
             }
 
             Interlocked.Add(ref _sizeOfUnflushedTransactionsInJournalFile, totalPages);
-            if (tx.IsLazyTransaction)
+            if (tx.IsLazyTransaction == false)
```

Comment:
Okay, I think that I found it.. this call is supposed to force the flush after a lazy transaction, right?